build/datagen-steps/meta.js

```diff
@@ -0,0 +1,77 @@
+import fs from "fs";
+import path from "path";
+import { marked } from "marked";
+
+marked.setOptions({
+  renderer: new marked.Renderer(),
+  gfm: true,
+  tables: true,
+  breaks: false,
+  pedantic: false,
+  sanitize: false,
+  smartLists: true,
+  smartypants: false,
+});
+
+function isFileMarkdown(filePath) {
+  return path.basename(filePath).split(".")[1] === "md";
+}
+
+function filenameToHTML(filePath) {
+  return `${path.basename(filePath).split(".")[0]}.html`;
+}
+
+/**
+ * Convert meta markdown files to HTML.
+ *
+ * Never throws, all errors are caught.
+ *
+ * @param {Object} options
+ * @param {string} options.inputBases The base paths from which meta markdown files will be read
+ * @param {string} options.outputBase The base path to which converted HTML files will be written
+ */
+export default async function datagenMeta({ inputBases, outputBase }) {
+  await Promise.all(
+    inputBases.map(async (folder) => {
+      // Read markdown meta directory
+      let files;
+      try {
+        files = await fs.promises.readdir(folder);
+      } catch (err) {
+        console.error(`Error reading meta directory ${folder}: ${err.message}`);
+      }
+
+      await Promise.all(
```

Comment:
```suggestion
      return Promise.all(
```

Coupled with `flatMap` above, this will just await all the promises at once rather than looping through the folders

Reply:
Unless I'm wrong in my understanding of promises, which is also totally possible!

Reply:
Got it, that makes sense!